components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/dto/ApplicationKeyGenerateRequestDTO.java

```diff
@@ -36,6 +36,12 @@
   
   
   private List<String> scopes = new ArrayList<String>();
+  
+  
+  private String clientId = null;
+  
```

Comment:
Shall we remove additional new lines?

Reply:
This code is auto generated using swagger2cxf-maven-plugin. Is it ok if we remove it manually.

Reply:
Sorry. No need.